### DISCUSSION TITLE
Is there a support for accepting payments through TERMINAL READERS ? https://stripe.com/docs/api/terminal/readers

### DISCUSSION THREAD

**Discussion by yogikorke** [flutter-stripe/flutter_stripe#598] [Q&A]
Is there a support for accepting payments through terminal readers ? 
https://stripe.com/docs/api/terminal/readers

**Comment by Yahllil** [comment]
I know they are working on a Flutter version for it, but no one knows when it will be released.
I ended up implementing it myself in my Flutter project. If you need help with this feel free to contact me

**Reply by arthurauffray** [reply]
@Yahllil sounds interesting do you plan on sharing it to github? Thanks

**Comment by remonh87** [comment]
we are waiting from Stripe to launch a SDK version. Our aim is to follow their audited sdk implementation  and build a lightweight Dart wrapper around it. I asked Stripe again about the status but there is not a clear date set at the moment.  Great @Yahllil  that you implemented a solution. Can you share a bit more?

**Reply by Yahllil** [reply]
Sure. I've implemented a method channel that calls Stripe Terminal SDK.
It still needs some work, but overall it's working as expected
